src/redux/actions/instructorActions.js

```diff
@@ -30,22 +31,22 @@ export const getusers = () => async dispatch => {
   } catch (error) {
     dispatch({
       type: ERROR_ACTION,
-      payload: console.log(error),
```

Comment:
Nice job updating this so error message can be set to state instead of just logged to the console